src/chapters/setup-intro-data-types/setup.rst

```diff
@@ -1,2 +1,218 @@
 Setup For Java
 ===============
+
+For the entirety of this course, we will be coding in Java. We need to not only have
+Java installed on our machines, but also some accompanying technologies to allow
+us to run and edit our Java code.
+
+Java Development Kit
+--------------------
+
+Installing Java means downloading a package of software called the **Java Development Kit**,
+or **JDK**, for short. The JDK contains software the tools needed to develop and
+run Java code, namely the the Java compiler, **javac**, and the 
+**Java Virtual Machine (JVM)**. 
+
+While the compiler is responsible for processing Java code into machine readable
+code, the JVM allows us to run that code on any computer. These tools 
+together, downloaded as the JDK, give us the means to write, compile, and run Java
+on our machines.
+
+A step-by-step walk-through of the process:
```

Comment:
The "step-by-step" reference implies a list.  How about converting the following items into an ol?